### ISSUE TITLE
FileNotFoundException When loading PDF from assets/subfolder

### ISSUE THREAD

**Issue by TheMaxCoder** [barteksc/AndroidPdfViewer#65]
Amazing lib, but am having strange issue, its throwing `FileNotFoundException`  when I try to load any PDF from any sub folder in assets directory however it works fine if I move that folder in assets directory.

Here is the `Logcat`:
 
```
load pdf error
java.io.FileNotFoundException: /data/data/com.xxx/cache/MySubFolder/MyPdf.pdf-pdfview.pdf: open failed: ENOENT (No such file or directory)
    at libcore.io.IoBridge.open(IoBridge.java:409)
    at java.io.FileOutputStream.<init>(FileOutputStream.java:88)
    at java.io.FileOutputStream.<init>(FileOutputStream.java:73)
    at com.github.barteksc.pdfviewer.util.FileUtils.copy(FileUtils.java:41)
    at com.github.barteksc.pdfviewer.util.FileUtils.fileFromAsset(FileUtils.java:34)
    at com.github.barteksc.pdfviewer.DecodingAsyncTask.doInBackground(DecodingAsyncTask.java:59)
    at com.github.barteksc.pdfviewer.DecodingAsyncTask.doInBackground(DecodingAsyncTask.java:30)
    at android.os.AsyncTask$2.call(AsyncTask.java:287)
    at java.util.concurrent.FutureTask.run(FutureTask.java:234)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1080)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:573)
    at java.lang.Thread.run(Thread.java:841)
Caused by: libcore.io.ErrnoException: open failed: ENOENT (No such file or directory)
    at libcore.io.Posix.open(Native Method)
    at libcore.io.BlockGuardOs.open(BlockGuardOs.java:110)
```

Thanks

**Comment by danielpassos** [comment]
I have the same problem. Do you guys have some plan to fix it?

**Comment by barteksc** [comment]
Yes, I will fix that soon.

**Comment by barteksc** [comment]
Fixed in version 2.1.0.

**Comment by danielpassos** [comment]
@barteksc Tested and works like a charm. Thanks!
